designer/server/src/types.ts

```diff
@@ -14,15 +12,4 @@ declare module '@hapi/hapi' {
   interface Server {
     logger: Logger
   }
-}
-
-declare module '@hapipal/schmervice' {
-  interface RegisteredServices {
-    persistenceService: PersistenceService
-  }
-
-  interface SchmerviceDecorator {
-    (all?: boolean): RegisteredServices
-    (namespace?: string[]): RegisteredServices
-  }
-}
+}
```

Comment:
Reminds me we need an **.editorconfig**

Reply:
Resolved by https://github.com/DEFRA/forms-designer/pull/87